content/en/docs/tasks/administer-cluster/cpu-management-policies.md

```diff
@@ -4,12 +4,14 @@ reviewers:
 - sjenning
 - ConnorDoyle
 - balajismaniam
+
 content_type: task
+min-kubernetes-server-version: v1.26
```

Comment:
In the prerequisites section, consider mentioning that if anyone is using an older version of Kubernetes, they should look at the docs for the version they are actually running.